lib/presentation/model/file/presentation_file.dart

```diff
@@ -42,4 +42,6 @@ abstract class PresentationFile {
   int fileSize();
 
   Widget fileIcon();
+
+  String fileIconImagePath();
```

Comment:
why we need this method?

Reply:
It is not needed and i removed it. Thank you, please